codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -204,7 +226,7 @@ public void sendFlowControlled(ChannelHandlerContext ctx, Http2Stream stream, Fl
             frame.error(t);
             return;
         }
-        state.writeBytes(state.writableWindow());
+        // We do not write bytes yet, instead bytes are written later by calling flush.
```

Comment:
Should we rename this method to something like `addFlowControlled` to indicate that it's buffering?